### PR TITLE
Incidents

### LINKED ISSUE #7: GET Open Incidents

### LINKED ISSUE #8: GET Resolved Incidents

### PR DESCRIPTION
**Pull request by amsmyth1**
### What Changed?
- Complete endpoint for incidents that are active and inactive
- Add script command to .travis.yaml to try to trigger db migration and seed with CI
- Change wording to active rather than open
### What's Next?
- Send updated API Contract
### Known Issues?
- None
### Test Coverage?
- 100% Sad, Happy, and Edge cases tested for Incidents
### README & Documentation
- Updated the endpoint URI and json responses for active and inactive incidents
### Closed Issues
close #8 
close #7

### CODE REVIEW

**Review by jmichlitsch — APPROVED** [review]
Good work on these, hoping that Travis will run those rake tasks properly this time around.

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ before_install:
 script:
 - bundle exec rails db:{create,migrate} RAILS_ENV=test
 - bundle exec rake db:schema:load
-- bundle exec rake csv_load:all
+- bundle exec rake db:test:prepare
```
Comment (klatour324) [inline .travis.yml:12]: Ahh yes! We came across this exact script in our researching for the other endpoint and getting it to deploy. Let's see if this works and loads up all the csv data through Travis CI

**Review by klatour324 — APPROVED** [review]
Looks great Alexa! I saw you updated the status from open to active which is great. If it hasn't been updated already, I will make a reminder note to do so in our db schema designer as well. Nice work!
